config/development.js

```diff
@@ -0,0 +1,14 @@
+const DYNAMODB_ENDPOINT = "http://localhost:8000";
```

Comment:
this should be dynamic based on the serverless.yml config

Reply:
You can define environment variables in serverless based on stage, function, or use globals, so we could use a `process.env.DYNAMODB_ENDPOINT` to get this per environment.  We could probably get rid of `config/development` in favor of a general config and utilize serverless further.